index.html

```diff
@@ -206,6 +206,20 @@ <h2>Skills</h2>
 
     </div>
   </section>
+
+  <section class="form-background">
+    <h1 class="form-heading">Contact me</h1>
+    <p class="form-content">If you have an application you are interested in developing, a feature that you need built
+      or a project that needs coding. I’d love to help with it.</p>
+    <form class="contact-form" action="https://formspree.io/f/mvolvljr" method="POST">
+      <input class="name" type="text" name="firstname" maxlength="30" placeholder="Please enter your name " required>
+      <input class="email" type="email" name="email" placeholder="Please enter your email" required>
+      <textarea class="message" name="message" maxlength="500" placeholder="Write your message here"
+        required></textarea>
+      <button class="submit" type="submit">Get it touch</button>
+    </form>
+  </section>
```

Comment:
- [ ] Kindly remove the extra white space to the left and right of your design to match the Figma template.

![image](https://user-images.githubusercontent.com/62290545/186463086-8896bc61-1364-4afe-8f04-ba081af2976e.png)